view/frontend/web/js/checkout-button.js

```diff
@@ -157,6 +158,26 @@ function ($, Component, placeAmwalOrder, payAmwalOrder, amwalErrorHandler, urlBu
                 attributes: false,
                 characterData: false
             });
+        },
+        /**
+         * Initialize login specific actions
+         */
+        initializeLogin: function () {
+            let self = this;
+            /**
+             * Check if the cart is empty
+             * @return Boolean
+             */
+            const isCartEmpty = () => {
+                const cart = customerData.get('cart');
+                return !cart().summary_count;
+            }
+            const loginButton = document.querySelector('.amwal-express-checkout-button.login');
+            const or = document.querySelector('.amwal-express-checkout-or');
+            if (!isCartEmpty()) {
```

Comment:
Is it correct that we show the button only if the cart is not empty?

Reply:
Yes, it's like the mini cart, we just show the button if the cart empty.